### PR TITLE
Upgrades generate_dev_bundle scripts on both Unix and Windows to use npm 3

### PR DESCRIPTION
**Pull request by martijnwalraven**
Only uses npm 3 for generating the bundles, we'll keep using npm 1.x for everything else for now.

### CODE REVIEW

File: tools/files.js [hunk 1]
```diff
@@ -643,7 +642,7 @@ files.freeTempDir = function (tempDir) {
       // Don't crash and print a stack trace because we failed to delete a temp
       // directory. This happens sometimes on Windows and seems to be
       // unavoidable.
-      Console.debug(err);
```
Comment (stubailo) [inline tools/files.js]: Why this change?

Comment (martijnwalraven) [inline tools/files.js]: I was pairing with Slava yesterday and we got a Console is undefined error that we didn't understand. The calls happen in a weird context (responding to a signal and clearing up), so that may have something to do with it (although I have no idea what that would do).
